src/Sentry/buildTransitive/Sentry.props

```diff
@@ -32,4 +32,69 @@
+    <SentryCLIOptions Condition="'$(SentryProject)' != ''">$(SentryCLIOptions) --project '$(SentryProject)'</SentryCLIOptions>
+  </PropertyGroup>
+
+  <Target Name="_PrepareSentryCLI" AfterTargets="DispatchToInnerBuilds;AfterBuild" Condition="'$(UseSentryCLI)' == 'true'">
+    <!-- Sentry CLI comes from the Sentry Nuget package when installed. -->
+    <PropertyGroup Condition="'$(SentryCLIDirectory)' == ''">
+      <SentryCLIDirectory Condition="'$(PkgSentry)' != ''">$(PkgSentry)\tools\</SentryCLIDirectory>
+    </PropertyGroup>
+
+    <!--
+      Choose the correct Sentry CLI executable depending on OS platform and architecture.
+      For Windows on Arm64, we'll use the X64 build for now (which should run via emulation).
+      Switch to a Windows Arm64 build when available. See https://github.com/getsentry/sentry-cli/issues/1426
+    -->
+    <PropertyGroup Condition="'$(SentryCLI)' == '' And '$(SentryCLIDirectory)' != ''">
+      <_OSArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</_OSArchitecture>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('OSX')) And $(_OSArchitecture) == 'Arm64'">$(SentryCLIDirectory)sentry-cli-Darwin-arm64</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('OSX')) And $(_OSArchitecture) == 'X64'">$(SentryCLIDirectory)sentry-cli-Darwin-x86_64</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Linux')) And $(_OSArchitecture) == 'Arm64'">$(SentryCLIDirectory)sentry-cli-Linux-aarch64</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Linux')) And $(_OSArchitecture) == 'X86'">$(SentryCLIDirectory)sentry-cli-Linux-i686</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Linux')) And $(_OSArchitecture) == 'X64'">$(SentryCLIDirectory)sentry-cli-Linux-x86_64</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Windows')) And $(_OSArchitecture) == 'Arm64'">$(SentryCLIDirectory)sentry-cli-Windows-x86_64</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Windows')) And $(_OSArchitecture) == 'X86'">$(SentryCLIDirectory)sentry-cli-Windows-i686</SentryCLI>
+      <SentryCLI Condition="$([MSBuild]::IsOSPlatform('Windows')) And $(_OSArchitecture) == 'X64'">$(SentryCLIDirectory)sentry-cli-Windows-x86_64</SentryCLI>
+      <SentryCLI Condition="!Exists('$(SentryCLI)')"/>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <SentryCLIInfo Include="Sentry CLI Configuration:" />
+    </ItemGroup>
+    <Exec Command="'$(SentryCLI)' info" Condition="'$(SentryCLI)' != ''" IgnoreExitCode="true" ConsoleToMsBuild="true" StandardOutputImportance="Low">
+      <Output TaskParameter="ExitCode" PropertyName="_SentryCLIStatusCode" />
+      <Output TaskParameter="ConsoleOutput" ItemName="SentryCLIInfo" />
+    </Exec>
+
+    <Message Importance="High" Condition="'$(_SentryCLIStatusCode)' != '0'" Text="The Sentry CLI is not fully configured with authentication, organization, and project." />
```

Comment:
I've been debating whether this should be a message or a warning.  If it were a warning, then right after updating to the version that includes this, people will start getting the warning in release builds.  That would be good from the perspective that they now have signal to either go configure their auth, org, and project, or to disable symbol upload with `<SentryUploadSymbols>false</SentryUploadSymbols>`.  But it would also be bad from the perspective that some people treat warnings as errors and we'd break their build.  Also if they disable symbol upload now, they might never turn it back on.

So for now I think we stick with an informational message.  We can consider switching to warning later, perhaps.

Reply:
Agree with the thought process. And turning into a Warning could go on next major.
But the `!= 0` is the right thing? Do we have an error code specific for missing auth or (different one?) org or project?

I wonder if we'd be giving this message if Sentry CLI was just erroring out for some other reason (can't reach Sentry right now).

Reply:
Oh, actually I need to do it slightly different if org and project are configured manually.  (It has a `--no-defaults` option to only check the authentication part.)

But also, yes it appears we need to check the actual output to determine if the Sentry server is unreachable.  The exit code alone is insufficient.  (It returns 1.)
Should the build warn/error in such a case?

Reply:
I believe I've addressed this with the latest changes. Thanks.